src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -108,8 +108,7 @@ public CreateGameController(FafService fafService, MapService mapService, ModSer
   }
 
   public void initialize() {
-    mapPreviewPane.minHeightProperty().bind(mapPreviewPane.widthProperty());
-    mapPreviewPane.maxHeightProperty().bind(mapPreviewPane.widthProperty());
+    mapPreviewPane.prefHeightProperty().bind(mapPreviewPane.widthProperty());
```

Comment:
Is that what we want,? I guess this was meant to avoid resizing the window  🤔

Reply:
The problem with the implementation before was, that the width can get bigger than the space left in height which means the map will overgrow because the height was bound to minHeight.. Binding to prefHeight avoids that.